src/kibana/components/vislib/visualizations/tile_map.js

```diff
@@ -404,25 +512,26 @@ define(function (require) {
     };
 
     /**
-     * zoomScale uses map zoom to determine a scaling
-     * factor for circle marker radius to display more
-     * consistent circle size as you zoom in or out
+     * geohashMinDistance returns a min distance in meters for sizing
+     * circle markers to fit within geohash grid
      *
-     * @method radiusScale
-     * @param value {Number}
-     * @param max {Number}
+     * @method geohashMinDistance
+     * @param feature {Object}
      * @return {Number}
      */
-    TileMap.prototype.zoomScale = function (zoom) {
-      var zScale = d3.scale.pow()
-        .exponent(4)
-        .domain([1, 12])
-        .range([0.3, 100]);
-      return zScale(zoom);
+    TileMap.prototype.geohashMinDistance = function (feature) {
+      var gh = feature.properties.rectangle;
+      var nw = L.latLng([gh[3][1], gh[3][0]]);
+      var ne = L.latLng([gh[2][1], gh[2][0]]);
+      var sw = L.latLng([gh[0][1], gh[0][0]]);
+      var ew = Math.floor(nw.distanceTo(ne));
+      var ns = Math.floor(nw.distanceTo(sw));
```

Comment:
two letter variables are hard to read, can you expand these to be descriptive of their use?